### ISSUE TITLE
[IAM] AmazonIdentityManagement with null message is thrown instead of EntityAlreadyExistsException

### ISSUE THREAD

**Issue by Ghilteras** [localstack/localstack#2231]
# Type of request: This is a ...

[X] bug report

# Detailed description
`EntityAlreadyExistsException` is not thrown correctly when creating IAM objects that are already present. `AmazonIdentityManagementException` with a null message is thrown instead

## Expected behavior
Localstack should throw `EntityAlreadyExistsException` with a populated message (not null)

## Actual behavior
```
com.amazonaws.services.identitymanagement.model.AmazonIdentityManagementException: null (Service: AmazonIdentityManagement; Status Code: 409; Error Code: 409 Conflict; Request ID: null)
```

# Steps to reproduce
- create an IAM role
- try to re-create it, catch `EntityAlreadyExistsException` but `AmazonIdentityManagementException` with null message is thrown instead

## Command used to start LocalStack
docker-compose up with `0.10.9`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
```
try {
    localStackIAMClient.createRole(createRoleRequest);
    localStackIAMClient.createRole(createRoleRequest);
} catch (EntityAlreadyExistsException e) {
    // AmazonIdentityManagementException with null is thrown instead
}
```

**Comment by whummer** [comment]
Thanks for reporting @Ghilteras . This appears to be working in the `latest` version of the Docker image, an integration test has been added in #2245. Can you please give it another try with latest and confirm whether the issue still persists there? Thanks!

**Comment by Ghilteras** [comment]
@duongpv7 @whummer I just tested this on https://hub.docker.com/layers/localstack/localstack/latest/images/sha256-e75c350ff04c19c5736ad19a48e426998ab152c0128a84d225c0bb07a34a7555?context=explore and the error is still present.

**Comment by Ghilteras** [comment]
@whummer @duongpv7 I opened another issue because I cannot re-open this

**Comment by whummer** [comment]
Thanks @Ghilteras , we were able to reproduce this issue - it should now finally be fixed in #2316 . A test has been added in the Java libs here: https://github.com/localstack/localstack-java-utils/blob/master/src/test/java/cloud/localstack/docker/BasicDockerFunctionalityTest.java#L209-L221 . Please give it a try with the `latest` version and let us know if the problem persists. Thanks

**Comment by Ghilteras** [comment]
@whummer when are you guys going to build the next release containing this fix?

**Comment by Ghilteras** [comment]
@whummer is this released as part of 0.11.1 ?
